test/extended/prometheus/prometheus.go

```diff
@@ -195,7 +195,7 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			}
 			runQueries(tests, oc, ns, execPodName, url, bearerToken)
 		})
-		g.It("should report less than two alerts in firing or pending state", func() {
+		g.It("should report less than two alerts in firing or pending state [IgnoreInDR]", func() {
```

Comment:
Why are alerts firing after you recover?

Reply:
This one is flaking if this test randomly start in the beginning of the suite - usually its either crashlooping openshift-apiserver or kubelet alert

Reply:
I think in the jobs we'll just create a wait of 1-3 minutes before the e2es kick off.  So you can remove this bit of code (excluding these tests) and remove the DR suite in cmd/openshift-tests.

Reply:
I'd rather do this in a follow up PR, does that sound good?

Reply:
No, I don't think these labels should be added at all, and I'm trying to get you to remove your suite so you use the disruptive suite as we intended.

Reply:
Turns out its a regression in 4.1.x (and in 4.2 as test shows) - https://bugzilla.redhat.com/show_bug.cgi?id=1731827